bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/ResourceBundleTracker.java

```diff
@@ -30,16 +31,15 @@
  * @author Markus Rathgeb - Add locale provider support
  * @author Ana Dimova - fragments support
  */
-@SuppressWarnings("deprecation")
-public class ResourceBundleTracker extends ResolvedBundleTracker {
+@SuppressWarnings({ "deprecation", "rawtypes", "unchecked" })
```

Comment:
IIRC we already discussed the "omit supress warnings for the whole class if possible" already.

Isn't it possible here?